### PR TITLE
feat(swb-ui): remove all SWB v2 UI

### PR DESCRIPTION
**Pull request by kpark277**
Issue #, if available:

Description of changes:
* All UI components for SWB v2 will be removed from solutions

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?
* [x] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: solutions/swb-reference/src/constants.ts [hunk 1]
```diff
@@ -40,7 +39,6 @@ interface Constants {
   MAIN_ACCT_ALB_ARN_OUTPUT_KEY: string;
   SWB_DOMAIN_NAME_OUTPUT_KEY: string;
   MAIN_ACCT_ALB_LISTENER_ARN_OUTPUT_KEY: string;
-  ECR_REPOSITORY_NAME_OUTPUT_KEY: string;
   VPC_ID_OUTPUT_KEY: string;
   ALB_SUBNET_IDS: string[];
```
Comment (DaveMadsen) [inline solutions/swb-reference/src/constants.ts:43]: still need ALB here?

File: solutions/swb-reference/src/SWBStack.ts [hunk 2]
```diff
@@ -115,7 +113,6 @@ export class SWBStack extends Stack {
       MAIN_ACCT_ALB_ARN_OUTPUT_KEY,
       SWB_DOMAIN_NAME_OUTPUT_KEY,
       MAIN_ACCT_ALB_LISTENER_ARN_OUTPUT_KEY,
-      ECR_REPOSITORY_NAME_OUTPUT_KEY,
       VPC_ID_OUTPUT_KEY,
       ALB_SUBNET_IDS,
```
Comment (DaveMadsen) [inline solutions/swb-reference/src/SWBStack.ts:117]: consider double checking if ALB and related infrastructure components are still needed.

**Review by DaveMadsen — APPROVED** [review]
I don't think there is anything blocking here. You may want to go back to `swb-reference/src` and double check `SWBStack.ts` and `constants.ts` to ensure you removed all of the UI related infrastructure and references. I noted you still have values around for the ALB which I thought was part of the UI components.

**Review by SanketD92 — APPROVED** [review]

File: solutions/swb-reference/src/constants.ts (hunk 1 above)
Comment (kpark277) [reply solutions/swb-reference/src/constants.ts:43]: Yes, ALB will still be used for allowing BYON at a backend level and that feature will not be removed. Only BYON from a UI standpoint.

**Pull request merged**